tests/sps/validation/test_article_citations.py

```diff
@@ -64,8 +65,8 @@ def test_validate_article_citation_year_success(self):
             </article>
         """
 
-        data = etree.fromstring(xml)
-        obtained = list(ArticleCitationsValidation(data).validate_article_citation_year())
+        data = list(ArticleCitations(etree.fromstring(xml)).article_citations)[0]
```

Comment:
@Rossi-Luciano ao adicionar `[0]` vc está assumindo que o valor retornado é 0 ou 1, e ser for mais por "erro"?